pytradfri/const.py

```diff
@@ -129,3 +129,6 @@
 SUPPORT_HEX_COLOR = 4
 SUPPORT_RGB_COLOR = 8
 SUPPORT_XY_COLOR = 16
+
+OBSERVATION_SLEEP_TIME = 5
+OBSERVATION_TIMEOUT = 10
```

Comment:
I'm still not sold on these values, if I don't touch any of my lights for a few hours they won't send any events IIRC. Meaning that the `_check_observations` function taking a guess at best, and it'll consider all my observations to be 'dead' when I make a request.

I'm cautious as many users (such as myself) have good stability on observations, I can have connections open for weeks without issue. This change potentially closes observations unnecessarily.

Reply:
That behaviour should never happen though. In my testing it still hasn't. Even after a few hours or close to a day, the observations don't just reset. They just work. What happens is the following flow:

 1. You send a command to the Tradfri gateway
 2. The check method for dead observations is triggered
    1. This method sets checking = True
    2. This method waits for 5 seconds initially
    3. **go to 3 below**
    4. The last updated global timer is checked against the values defined in const
        1. If the time is above the defined times, the observations will be reset
        2. If the time is below the defined times, nothing happens
    5. Set checking = False
 3. The Tradfri hub sends a response in the observation callback
    1. The state of the object is changed from the callback
    2. The last updated global time is updated
    3. **go to 2.4 above**

In my experience most of this happens in a fraction of a second. Almost all of my requests were sub second response times. In some edge cases it took a bit longer, but not that long. We can always up this value though. But I encourage you to test it out for yourself, perhaps with debug logging like:

```python
_LOGGER.debug('Tradfri response_time %f', time.time() - APIFactory.get_last_changed())
```

Reply:
If the device responds, the observation callback will be triggered. It's the same as before.

But if no response from the Tradfri gateway will be received within the timeout window, the observations will be reset. That's the only functional change.